assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -135,6 +153,10 @@ function ExecutionResults({
         description: getCheckDescription(catalog, checkID),
         expectationResults,
         agentsCheckResults: addHostnameToTargets(agentsCheckResults, hostnames),
+        onClickRemediation: () => {
```

Comment:
`onClickRemediation` looks a bit confusing?
Maybe simply `onClick` or `onClickCheckID`.
First of all, because we don't only show the remediation and seconds `Remediation` doesn't match what the user does, which is to click in the check id field

Reply:
Fair point, naming is always hard hehe :D